api/controller.py

```diff
@@ -304,11 +304,14 @@ def get_domain(url):
         authentication_document_cache_time = int(
             ConfigurationSetting.sitewide(
                 self._db, Configuration.AUTHENTICATION_DOCUMENT_CACHE_TIME
-            ).value_or_default(3600)
+            ).value_or_default(0)
```

Comment:
Is there a way to completely skip the cache instead of setting the retention time span to 0? There might be a performance impact to skip it altogether (although I have no profiled the run time for this). This question is moot if the cache is an intended/required function of the architecture.

Reply:
I could remove the cache altogether without changing much, but conceptually it makes a lot of sense, so I'd like to keep the code in place and fix it (or decide to remove it) later.